packages/terra-form-input/src/Input.module.scss

```diff
@@ -6,22 +6,26 @@
     background-repeat: no-repeat;
     background-size: var(--terra-form-input-focus-background-start);
     border: var(--terra-form-input-border, 1px solid #dedfe0);
-    border-radius: var(--terra-form-input-border-radius, 0.2rem);
+    border-radius: var(--terra-form-input-border-radius, 3px);
     box-shadow: var(--terra-form-input-box-shadow);
-    color: var(--terra-form-input-color, #000);
+    color: var(--terra-form-input-color, #1c1f21);
     font-family: inherit;
-    font-size: var(--terra-form-input-font-size, 1.143rem);
-    height: var(--terra-form-input-height, 2.143rem);
+    font-size: var(--terra-form-input-font-size, 1.142857rem);
+    height: var(--terra-form-input-height, 2.142857rem);
     line-height: var(--terra-form-input-line-height, 1.25);
-    padding-bottom: var(--terra-form-input-padding-bottom, 0.286rem);
-    padding-left: var(--terra-form-input-padding-left, 0.286rem);
-    padding-right: var(--terra-form-input-padding-right, 0.286rem);
-    padding-top: var(--terra-form-input-padding-top, 0.286rem);
+    padding-bottom: var(--terra-form-input-padding-bottom, 0.28571rem);
+    padding-left: var(--terra-form-input-padding-left, 0.28571rem);
+    padding-right: var(--terra-form-input-padding-right, 0.28571rem);
+    padding-top: var(--terra-form-input-padding-top, 0.28571rem);
     transition-duration: var(--terra-form-input-transition-duration, 0s);
     transition-property: background-size;
     transition-timing-function: var(--terra-form-input-transition-timing-function, ease);
     width: 100%;
 
+    &::placeholder {
+      color: var(--terra-form-input-placeholder-color, #70787c);
```

Comment:
The theme I've been working calls out setting the placeholder text to be italic. Unless UX has any objections could we also add a variable here for `font-style`?

cc: @neilpfeiffer

Reply:
Correct. @amichaelparker could you add a new line for `font-style` with a themeing variable with no value set for default for placeholders:

```
font-style: var(--terra-form-input-placeholder-font-style);
```

\
\
Thanks @StephenEsser for the catch.

Reply:
Time input is grabbing the theming values from input as we are composing the terra-form-input to generate the time input. I'd say let's not make theming variables for placeholder for time input as the ones used in terra-form-input will be used in time input with the current composition of these components.